include/inviwo/core/properties/optionproperty.h

```diff
@@ -193,14 +204,19 @@ class TemplateOptionProperty : public BaseOptionProperty {
     TemplateOptionProperty& replaceOptions(const std::vector<T>& options);
 
     virtual bool isSelectedIndex(size_t index) const override;
-    virtual bool isSelectedIdentifier(const std::string& identifier) const override;
-    virtual bool isSelectedDisplayName(const std::string& name) const override;
+    virtual bool isSelectedIdentifier(std::string_view identifier) const override;
+    virtual bool isSelectedDisplayName(std::string_view name) const override;
     bool isSelectedValue(const T& val) const;
 
     const T& get() const;
     const T& operator*() const;
     const T* operator->() const;
-    void set(const T& value);
+
+    bool set(const T& value);
```

Comment:
Return true if modified?

Reply:
it returns true if it was actually set, i.e. if the value was found in the list of options

Reply:
will add a comment...